src/btree/bt_split.c

```diff
@@ -2304,6 +2304,13 @@ __wt_split_rewrite(WT_SESSION_IMPL *session, WT_REF *ref, WT_MULTI *multi)
         F_SET_ATOMIC_16(page, WT_PAGE_EVICT_NO_PROGRESS);
     __wt_ref_out(session, ref);
 
+    /*
+     * It is rare to rewrite an internal page. However, when that happens, we need to update the
+     * parent ref on the page as we used a dummy ref to instantiate the page.
+     */
+    if (F_ISSET(ref, WT_REF_FLAG_INTERNAL))
```

Comment:
Could you add a statistic for this please Chenhao - I'd be interested to know if it happens in the field (though I guess it results in a crash if this code isn't there?)

Is a test case possible? Does it require special configuration options to scrub?

Reply:
A test case is hard. We need to add some configuration options to scrub.